vault/identity_store_entities.go

```diff
@@ -25,19 +25,19 @@ func entityPaths(i *IdentityStore) []*framework.Path {
 			Fields: map[string]*framework.FieldSchema{
 				"id": {
 					Type:        framework.TypeString,
-					Description: "ID of the entity",
+					Description: "ID of the entity. If set, updates the corresponding existing entity.",
 				},
 				"name": {
 					Type:        framework.TypeString,
 					Description: "Name of the entity",
 				},
 				"metadata": {
 					Type:        framework.TypeKVPairs,
-					Description: "Metadata to be associated with the entity.",
+					Description: "Metadata to be associated with the entity. Format should be a list of 'key=value' pairs.",
```

Comment:
I don't think this format string is correct.  API users should use map[string]string which should be the primary use case.